pkg/ruler/manager.go

```diff
@@ -20,6 +20,24 @@ import (
 	"github.com/cortexproject/cortex/pkg/util"
 )
 
+var (
+	configUpdatesTotal = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "cortex",
+		Name:      "ruler_config_updates_total",
+		Help:      "Total number of config updates triggered by a user",
+	}, []string{"user"})
+	configUpdateFailuresTotal = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "cortex",
+		Name:      "ruler_config_update_failures_total",
+		Help:      "Total number of config update failures triggered by a user",
+	}, []string{"user", "reason"})
+	userManagerFailed = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Instead of having global metrics, can we move them to DefaultMultiTenantManager? We already have managersTotal there.